src/packages/version-check/npm-shrinkwrap.json

```diff
@@ -0,0 +1,3126 @@
+{
```

Comment:
We use package-lock.json for the other packages. I'm thinking we should do that here, too. The original _intent_ of using package-lock.json in the `@ganache/*` packages was to give users who wanted to play fast and loose with transitive dependency versions could more easily do so.